src/noteburst/worker/nublado.py

```diff
@@ -86,10 +87,36 @@ async def spawn(
         )
 
         await nublado_client.auth_to_hub()
-        await nublado_client.spawn_lab(config=nublado_image)
+        logger.info("Auth'd to hub.")
```

Comment:
```suggestion
        logger.info("Authenticated to JupyterHub.")
```

might be a bit easier to search without escaping a `'`?